src/main/java/net/neoforged/neoforge/client/event/RegisterDimensionTransitionEvent.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.event;
+
+import java.util.Map;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.world.level.Level;
+import net.neoforged.bus.api.Event;
+import net.neoforged.bus.api.ICancellableEvent;
+import net.neoforged.fml.LogicalSide;
+import net.neoforged.fml.event.IModBusEvent;
+import net.neoforged.neoforge.client.DimensionTransitionScreen;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * <p>Event for registering screen effects when transitioning across dimensions.</p>
```

Comment:
This should clarify which has priority: the dimension transition screen of the originating dimension, or that of the destination dimension?